ivy/func_wrapper.py

```diff
@@ -40,13 +41,18 @@ def _get_first_array(*args, **kwargs):
             arr = ivy.index_nest(kwargs, arr_idxs[0])
     return arr
 
-
-# Array Handling #
+# Just a native python compile function with cache #
```

Comment:
Just write "Compiling with Cache", and change the number of `-` so that the title aligns correctly please :)